docs/examples/tf/tf_distributed/mnist_example.py

```diff
@@ -17,7 +17,9 @@ def train_dense_model(batch_size):
     tf.keras.backend.clear_session()  # For easy reset of notebook state.
 
     slurm_resolver = tf.distribute.cluster_resolver.SlurmClusterResolver(port_base=15000)
-    mirrored_strategy = tf.distribute.MultiWorkerMirroredStrategy(cluster_resolver=slurm_resolver)
+    communication = tf.distribute.experimental.CommunicationImplementation.NCCL
+    mirrored_strategy = tf.distribute.MultiWorkerMirroredStrategy(cluster_resolver=slurm_resolver, 
+                                                                  communication=communication)
```

Comment:
I think the kwarg name is `communication_options` for tf 2.4.0.
```suggestion
                                                                  communication_options=communication)
```

Reply:
Hi @zaccharieramzi , looking at the [source code](https://github.com/tensorflow/tensorflow/blob/v2.4.1/tensorflow/python/distribute/collective_all_reduce_strategy.py), it seems you are right. I was fooled by the [the TF 2.4.1 doc](https://www.tensorflow.org/api_docs/python/tf/distribute/experimental/MultiWorkerMirroredStrategy), that mentions `communication`...

Reply:
Yes it fooled me too before, so that's why I was wary of this